Library/Homebrew/cmd/help.rb

```diff
@@ -1,18 +1,17 @@
 HOMEBREW_HELP = <<-EOS
 Example usage:
+  brew search [TEXT|/PATTERN/]
```

Comment:
Probably, because IIR "pattern" in Ruby and `make`/CMake documentation usually refers to a file globbing pattern instead of a regex, and there are often areas that support both.